misc/test_site/mac_addrs/9a02571e8f03/module_config.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
@ewRedstone @InBrewJ I think the point of having different module_config.json files, with some of them being incomplete is to provide both "positive" and "negative" cases. What is the rationale of having all these files looking the same?

Reply:
This is because I need the configuration:

 "switch": {
      "enabled": true	      "enabled": true,
      "poe":{
        "enabled": true
      }

and the files need to tidying up because DAQ is a mess